ccdproc/tests/test_image_collection.py

```diff
@@ -642,16 +642,15 @@ def test_keyword_order_is_preserved(self, triage_setup):
         assert ic.keywords == ['file'] + keywords
 
     def test_sorting(self, triage_setup):
-        collection = image_collection.ImageFileCollection(location=triage_setup.test_dir,
-                                             keywords=['imagetyp',
-                                                       'filter',
-                                                       'object'])
+        collection = image_collection.ImageFileCollection(
+            location=triage_setup.test_dir,
+            keywords=['imagetyp', 'filter', 'object'])
```

Comment:
This is not a real change it just refactors the way the code is organized. It's still PEP8 compliant (according to pyflakes).